Data/Infrastructure/OJS.Data.Infrastructure/DateTimeValueConverter.cs

```diff
@@ -0,0 +1,41 @@
+namespace OJS.Data.Infrastructure;
+
+using Common.Utils;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using System;
+using System.Linq.Expressions;
+using static Common.GlobalConstants.EnvironmentVariables;
+
+public class DateTimeValueConverter : ValueConverter<DateTime?, DateTime?>
+{
+   private static readonly Expression<Func<DateTime?, DateTime?>> Serialize = date => ConvertToUtc(date);
+
+   private static readonly Expression<Func<DateTime?, DateTime?>> Deserialize = date => ConvertToLocal(date);
+
+   public DateTimeValueConverter()
+      : base(Serialize, Deserialize)
+   {
+   }
+
+   private static DateTime? ConvertToUtc(DateTime? dateTime)
+   {
+      if (dateTime == null)
+      {
+         return null;
+      }
+
+      return DateTime.SpecifyKind(dateTime.Value, DateTimeKind.Utc);
```

Comment:
Provide the source time zone from which to convert to UTC.

Reply:
I tried to use TimeZoneInfo.ConvertTimeToUtc(dateTime.Value, TimeZoneInfo), but this triggers an error which occurs with different saved dates (some of them UTC, some Local).

Reply:
What is the error and how can we fix it? Saved dates don't have time zone info in them, how can a saved date be UTC or Local, it's just a date..